src/vmshepherd/iaas/vm.py

```diff
@@ -29,8 +29,7 @@ class VmState(Enum):
 
 class Vm:
     """
-    Vm class definition.
-    Define API to implement in specific IaasDrivers like Openstack or EC2.
+    Virtual machine object.
```

Comment:
It is not an vm object untill one make instance of this class ;)

Reply:
I would argue... docs is about functionality and responsibility. I agree that it requires more details/improvement, but info that this is a class is useless.